### ISSUE TITLE
how to solve URL fetch failure of pretrained weights ?

### ISSUE THREAD

**Issue by moyangjiayou** [idealo/image-super-resolution#129]
Hello,
when I use rdn = RDN(weights='psnr-large'),I got the download URL fetch failure?
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    rdn = RDN(weights='psnr-large')
  File "/opt/conda/envs/singan/lib/python3.6/site-packages/ISR-2.2.0-py3.6.egg/ISR/models/rdn.py", line 108, in __init__
  File "/opt/conda/envs/singan/lib/python3.6/site-packages/tensorflow_core/python/keras/utils/data_utils.py", line 255, in get_file
    raise Exception(error_msg.format(origin, e.errno, e.reason))
Exception: URL fetch failure on https://public-asai-dl-models.s3.eu-central-1.amazonaws.com/ISR/rdn-C6-D20-G64-G064-x2/PSNR-driven/rdn-C6-D20-G64-G064-x2_PSNR_epoch086.hdf5: None -- [Errno 101] Network is unreachable

and I have tried load_weights functions to use local hdf5 weights , but got error of h5py ?
code:
rdn.model.load_weights('./image-super-resolution/weights/sample_weights/rdn-C6-D20-G64-G064-x2/ArtefactCancelling/rdn-C6-D20-G64-G064-x2_ArtefactCancelling_epoch219.hdf5')
error:
File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (file signature not found)

Then I tried git-lfs pull, I got error again...
batch response: This repository is over its data quota. Purchase more data packs to restore access. error: failed to fetch some objects from 'https://github.com/idealo/image-super-resolution.git/info/lfs'

so please help me~~~
how could I use the pretrained models? 
THANKS A LOT!

**Comment by cfrancesco** [comment]
Can you directly download the file from here? 
https://public-asai-dl-models.s3.eu-central-1.amazonaws.com/ISR/rdn-C6-D20-G64-G064-x2/PSNR-driven/rdn-C6-D20-G64-G064-x2_PSNR_epoch086.hdf5

I don't know why you get a network problem, if you can download it manually, use load weights with this file.

The other errors are to be expected, first is probably downloaded from git, which is not the real file, while the second is git-lfs simply being out of quota for downloads.

**Comment by moyangjiayou** [comment]
@cfrancesco Thanks for reply! 
But I can't directly download the file by url, I have tried to use wget and chrome, but I got 443 respond.
That's so sad.
wget https://public-asai-dl-models.s3.eu-central-1.amazonaws.com/ISR/rdn-C3-D10-G64-G064-x2/PSNR-driven/rdn-C3-D10-G64-G064-x2_PSNR_epoch134.hdf5
Resolving public-asai-dl-models.s3.eu-central-1.amazonaws.com (public-asai-dl-models.s3.eu-central-1.amazonaws.com)... 31.13.66.6, 2001::4a56:8e37
Connecting to public-asai-dl-models.s3.eu-central-1.amazonaws.com (public-asai-dl-models.s3.eu-central-1.amazonaws.com)|31.13.66.6|:443...
